### DISCUSSION TITLE
Complete result exceeds size limit (4442047 of 4194304)

### DISCUSSION THREAD

**Discussion by Amanraj0007** [airbytehq/airbyte#31956] [General]
am trying to sync data from postgres to snowflake
while syncing i got error Failure reason: `Something went wrong within the airbyte platform`
I got the error in worker’s container
`io.airbyte.commons.temporal.exception.SizeLimitException: Complete result exceeds size limit (4442047 of 4194304)`
I have raised the issue on github also , anyone  faced the similar issue?
["https://github.com/airbytehq/airbyte/issues/31954"](Github issue )

**Comment by marcosmarxm** [comment]
Duplicate of https://github.com/airbytehq/airbyte/issues/31954
